src/js/App/Sidenav/Navigation/ChromeLink.tsx

```diff
@@ -99,6 +104,11 @@ const LinkWrapper: React.FC<LinkWrapperProps> = ({ href, isBeta, onLinkClick, cl
     .join('_');
   return (
     <NavLink
+      onMouseEnter={() => {
+        if (moduleEntry) {
+          preloadModule(moduleEntry?.scope, moduleEntry?.module);
```

Comment:
I'd personally wait for some time before firing the preload, because if user hovers over entire nav they'd preload it. On other hand, this is applicable only to computers (and you'll usually have faster internet connection there) so it's good.

Reply:
Yup, @ryelo promised to provide some metricst. Lets just add a 100ms delay for now, before we have some data.